src/utils/usageData.ts

```diff
@@ -17,6 +17,9 @@ import {
 } from './persistentStore';
 
 const trackId = 'UA-22498474-5';
+const isDevelopment = process.env.NODE_ENV === 'development';
```

Comment:
This is the third way (at least) of checking if we are in development mode, I think we should agree on which we use :) I don't know which one is better, but we should definitely agree on what we want to use in the future.

Currently we are also using the `electron-is-dev` module in PPK and using `app.isPackaged` in `deviceLister.ts` also in shared.

Reply:
Sounds like a job for shared!

Reply:
Decided to go with app.isPackaged.